x-pack/plugins/infra/server/lib/metrics/lib/convert_histogram_buckets_to_timeseries.ts

```diff
@@ -20,6 +20,7 @@ import {
   PercentilesTypeRT,
   PercentilesKeyedTypeRT,
   TopHitsTypeRT,
```

Comment:
I wonder if we should remove all references to "top hits" if we don't plan to use it in the query anymore.

Reply:
I agree, I just wanted to double-check it wasn't used anywhere else. Done!